ocaml/xapi/xapi_pool_patch.ml

```diff
@@ -674,6 +674,7 @@ let apply ~__context ~self ~host =
 		error "Caught exception rescanning patch applied directory: %s" (ExnHelper.string_of_exn e)
 	    end;
           Create_misc.create_patches_requiring_reboot_info ~__context ~host;
+          Helpers.call_api_functions ~__context (fun rpc session_id -> Client.Client.Host.refresh_pack_info ~rpc ~session_id ~host);
```

Comment:
The `apply` function is already running on the right host (it has already been forwarded to a slave, if that was needed), so you don't need to go through the API. Also, even though `refresh_pack_info` does what you need, it is conceptually not the right call.